static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -93,32 +104,39 @@ type CrumbItemProps = {
 
 const CrumbItem = styled(PanelItem)<CrumbItemProps>`
   display: grid;
-  grid-template-columns: max-content max-content auto max-content;
-  align-items: center;
+  grid-template-columns: max-content auto;
+  align-items: flex-start;
   gap: ${space(1)};
   width: 100%;
 
   font-size: ${p => p.theme.fontSizeMedium};
   background: transparent;
-  padding: 0;
-  padding-right: ${space(1)};
+  padding: ${space(1)};
   text-align: left;
-
   border: none;
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-  ${p => p.isHovered && `background: ${p.theme.surface400};`}
-
-  /* overrides PanelItem css */
-  &:last-child {
-    border-bottom: 1px solid ${p => p.theme.innerBorder};
-  }
+  position: relative;
+  ${p => p.isSelected && `background-color: ${p.theme.purple100};`}
+  ${p => p.isHovered && `background-color: ${p.theme.surface100};`}
+  border-radius: 4px;
 
-  /* Selected state */
-  ::before {
+  &::after {
     content: '';
-    width: 4px;
+    position: absolute;
+    left: 19.5px;
+    width: 1px;
+    background: ${p => p.theme.gray200};
     height: 100%;
-    ${p => p.isSelected && `background-color: ${p.theme.purple300};`}
+    z-index: 1;
+  }
+
+  &:first-of-type::after {
+    bottom: 0;
+    height: 50%;
```

Comment:
instead of 50% we could be more specific and do like `top: 8px; bottom: 0`. The 8px isn't a magic number either, it's the same as the button padding.

This way if the crumb gets taller for whatever reason, the line will never start in the wrong spot.

Reply:
> Lets also make sure the height & margins of the crumbs are set to match the new numbers. Looks like placeholders are a little shorter than the real data.

Indeed this was a pretty good catch, now the placeholders look like this: 
![image](https://user-images.githubusercontent.com/39612839/178789531-fc11bdb1-6d75-4074-b880-7e77500c90a2.png)